README.md

```diff
@@ -46,24 +54,59 @@ The custom export format of JabRef is an alternative to the [Citation Style Lang
 - Author: Steffan Cook
 - File: [Adv_Mater.jstyle](Adv_Mater.jstyle)
 
+### American Journal of Hematology
+
+- Author: Katrina Vanura
+- File: [AmJHematol.jstyle](https://github.com/JabRef/jstyles.jabref.org/pull/5/commits/7152a3e702a66af02baf833c588ad73f1d3030c2)
+
 ### American Mineralogist
 
 - Author: Jan Stelling
 - File: [American_Mineralogist.jstyle](American_Mineralogist.jstyle)
 - ![preview](American_Mineralogist.png)
 
+### Annals of Hematology
+
+- Author: Katrina Vanura
+- File: [AnnHematol.jstyle](https://github.com/JabRef/jstyles.jabref.org/pull/4/commits/da4857a9f5c00c38be773c5959d7f8847c97e0e8)
```

Comment:
You just need to state the same "String" again:

    - File: [AnnHematol.jstyle](AnnHematol.jstyle)

The link is resolved to the same directory as the README.md. Browsers come along with it.